src/types/system.ts

```diff
@@ -43,17 +45,33 @@ export default /* GraphQL */ `
     pageInfo: PageInfo!
   }
 
-  "This type contains system-wise settings."
+  "This type contains system-wise info and settings."
   type Official {
     "Feature flag"
     features: [Feature!]!
+
+    "Announcements"
+    announcements(input: AnnouncementsInput!): [Announcement!]
   }
 
   type Feature {
     name: FeatureName!
     enabled: Boolean!
   }
 
+  type Announcement {
+    id: ID!
+    title: String
+    cover: String
+    content: String
+    link: String
+    type: AnnouncementType!
+    visible: Boolean!
+    order: Int!
```

Comment:
where's the `order` uses for? looks `official/announcements` don't order by `order`.

Reply:
It's a optional field for future implementation.